src/main/app/form/validation.ts

```diff
@@ -234,5 +257,8 @@ export const isMarkDownLinkIncluded: Validator = value => {
         return ValidationError.CONTAINS_MARKDOWN_LINK;
       }
     }
+    if (valueToValidate.includes('<') || valueToValidate.includes('>')) {
+      return ValidationError.CONTAINS_MARKDOWN_LINK;
```

Comment:
this isn't a markdown link - this is html? See validation used in update case for html tags

Reply:
Can this be changed and named accordingly please

Reply:
I'll check update case and include the method accordingly!